### ISSUE TITLE
Could not load file or assembly 'AutoMapper' or one of its dependencies after upgrade to 6.0.0.0

### ISSUE THREAD

**Issue by st-dev** [AutoMapper/AutoMapper#2012]
After upgrading our solution to the latest version 6.0.0.0 we are seeing the following exception. I'm not sure where the reference to 4.2.1.0 is coming from. The csproj file version changed to 0.0.0.0 not sure if thats related. 

thank you

---
`Exception Details: System.IO.FileLoadException: Could not load file or assembly 'AutoMapper' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)`

---

```
[HttpException (0x80004005): Could not load file or assembly 'AutoMapper, Version=4.2.1.0, Culture=neutral, PublicKeyToken=be96cd2c38ef1005' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)]
   System.Web.HttpRuntime.FirstRequestInit(HttpContext context) +9946132
   System.Web.HttpRuntime.EnsureFirstRequestInit(HttpContext context) +90
   System.Web.HttpRuntime.ProcessRequestNotificationPrivate(IIS7WorkerRequest wr, HttpContext context) +261
```

---

```
<Reference Include="AutoMapper, Version=0.0.0.0, Culture=neutral, PublicKeyToken=be96cd2c38ef1005, processorArchitecture=MSIL">
      <HintPath>..\..\packages\AutoMapper.6.0.0\lib\net45\AutoMapper.dll</HintPath>
      <Private>True</Private>
    </Reference>
```

**Comment by st-dev** [comment]
Same as issue #2010

**Comment by jbogard** [comment]
Fixed, pull 6.0.1.

**Comment by st-dev** [comment]
Thank you. 

It might be worth noting that with the extra dependecy on System.ValueTuple you need the following in web.config if using in MVC otherwise you get an exception in the view rendering. 

`		<compilation debug="true" targetFramework="4.5" defaultLanguage="c#">
            <assemblies>
              <add assembly="System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
            </assemblies>
		</compilation>`

**Comment by st-dev** [comment]
And thank you for the fix in 6.0.1
